app/code/Magento/Quote/etc/webapi_rest/di.xml

```diff
@@ -12,4 +12,7 @@
     <type name="Magento\Quote\Model\QuoteRepository">
         <plugin name="accessControl" type="Magento\Quote\Model\QuoteRepository\Plugin\AccessChangeQuoteControl" />
     </type>
+    <type name="Magento\Quote\Model\QuoteRepository">
+        <plugin name="authorization" type="Magento\Quote\Model\QuoteRepository\Plugin\Authorization" />
```

Comment:
Please add this plugin declaration to an existing `Magento\Quote\Model\QuoteRepository` type node in this file